receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -176,7 +176,7 @@ func (mg *metricGroupPdata) toDistributionPoint(orderedLabelKeys []string, dest
 	// The timestamp MUST be in retrieved from milliseconds and converted to nanoseconds.
 	tsNanos := pdataTimestampFromMs(mg.ts)
 	if mg.family.isCumulativeTypePdata() {
-		point.SetStartTimestamp(pdataTimestampFromMs(mg.intervalStartTimeMs))
+		point.SetStartTimestamp(pdataTimestampFromMs(mg.ts)) // metrics_adjuster adjusts the startTimestamp to the initial scrape timestamp
```

Comment:
Is the intervalStartTimeMs still used? Can you cache the `pdataTimestampFromMs(mg.ts)`?

Reply:
@bogdandrutu thanks. Updated. I have removed `intervalStartTimeMs` as it was no longer required. I have also used the cached `pdataTimestampFromMs(mg.ts)`